ircd/send.c

```diff
@@ -1199,6 +1198,38 @@ sendto_monitor(struct Client *source_p, struct monitor *monptr, const char *patt
 	msgbuf_cache_free(&msgbuf_cache);
 }
 
+/* sendto_monitor_with_capability_butserial()
+ *
+ * inputs	- monitor nick to send to, caps, negate caps, whether to send to clients having current serial, format, va_args
+ * outputs	- message to local users monitoring the given nick
+ * side effects -
+ */
+void
+sendto_monitor_with_capability_butserial(struct Client *source_p, struct monitor *monptr, int caps, int negcaps, bool skipserial, const char *pattern, ...)
```

Comment:
Eliminate current_serial abuse

```suggestion
sendto_monitor_with_capability(struct Client *source_p, struct monitor *monptr, int caps, int negcaps, const char *pattern, ...)
```